astropy/time/tests/test_basic.py

```diff
@@ -350,6 +356,35 @@ def test_creating_all_formats(self):
         Time(dt, format='datetime', scale='tai')
         Time([dt, dt], format='datetime', scale='tai')
 
+    def test_local_format_transforms(self):
+        """
+        Test trasformation of local time to different formats
+        Transformation to formats with reference time should give
+        ScalevalueError
+        """
+        t = Time('2006-01-15 21:24:37.5', scale='local')
+        assert t.jd == 2453751.3921006946
```

Comment:
I worry this may (start to) fail due to rounding errors on some platforms.  Maybe
```
assert_allclose(t.jd, 2453751.3921006946, atol=0.001/3600./24., rtol=0.)
```